### ISSUE TITLE
tagged-cmw-record

### ISSUE THREAD

**Issue by nedmsmith** [thomas-fossati/draft-ftbs-rats-msg-wrap#25]
Given a use case for an attestation results or other message that conveys other CMs that are relevant to a particular attestation result. Does it make sense to continue following the convention of using tagged cbor - as in:
```
tagged-cmw-record = #6.4711(bytes .cbor cmw-array)
```
A possible triple record structure that treats other CMs as triple objects might be:
```
ar-triple-record = [ 
     attester: stateful-environment-map,
     ar-claims: [ + $ar-claims-type-choice ] 
]
$ar-claims-type-choice /= tagged-cmw-record
```
The argument against is that cmw-array already has tagged values and the little bit of CDDL inside of `ar-claims:` isn't enough to warrant needing a CBOR tag. 
The argument for is that all `$ar-claims-type-choice` possibilities are expected to be `tagged-xxx` values that can be dispatched by a parser dispatcher following a consistent convention.

**Comment by nedmsmith** [comment]
Did this issue get resolved?

**Comment by thomas-fossati** [comment]
> Did this issue get resolved?

The question is who should be responsible for registering the tag:
* CMW itself, or
* the spec using CMW

**Comment by nedmsmith** [comment]
I think it would be convenient for this I-D to allocate a CBOR tag for `tagged-cmw-record`.
E.g.: `tagged-cmw-record = #6.TBA(cmw-record)`

**Comment by thomas-fossati** [comment]
quick note: this tag is to be used only for the CBOR array- and CBOR tag-based serialisations.

**Comment by thomas-fossati** [comment]
Fixed in #33
